src/Fixer/CastNotation/ModernizeTypesCastingFixer.php

```diff
@@ -37,7 +37,8 @@ public function getDefinition()
     $a = doubleval($b);
     $a = strval ($b);
     $a = boolval($b);
-'),
+'
+            ),
```

Comment:
I think so too, but should I do this manually or with the fixer. I kept it like that because I don't want to hide the fixer's shortcomings, and I want to know if this is supposed to be its responsability or not. If it is, I don't think it's going to be a really simple thing.